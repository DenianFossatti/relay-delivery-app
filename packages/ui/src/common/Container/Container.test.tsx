import { mockTestID, render, wrapper } from '../../tests';

import { Container, ContainerProps } from '.';

describe(`Container`, () => {
  const props: ContainerProps = {
    testID: 'Container',
  };

  const { findByTestId, toJSON } = render(<Container {...props} />, {
    wrapper,
  });

  const el_container = mockTestID('View', props.testID!);

  it(`render component #${el_container}`, async () => {
    const sut = await findByTestId(el_container);
    expect(sut).toBeTruthy();
  });

  it('should render snapshot', () => {
    expect(toJSON()).toMatchSnapshot();
  });
});
