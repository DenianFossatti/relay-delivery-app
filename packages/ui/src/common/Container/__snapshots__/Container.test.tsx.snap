// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/Container should render snapshot 1`] = `null`;
